### PR TITLE
backend/refactor/#966 remove static driver bpp module

### PR DESCRIPTION
**Pull request by Dwibedi-10**
## Type of Change

- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement
- [x] Refactoring
- [ ] Dependency updates

## Description
Removed static offer BPP module.

### Additional Changes

- [x] This PR modifies the database schema (database migration added)
- [x] This PR modifies dhall configs/environment variables
- [ ] This PR contains API breaking changes

## Checklist

- [x] I formatted the code and addressed linter errors `./dev/format-all-files.sh`
- [x] I reviewed submitted code
- [ ] I added unit tests for my changes where possible
- [ ] I added a [CHANGELOG](/CHANGELOG.md) entry if applicable

### CODE REVIEW

**Review by DonDokidon — CHANGES_REQUESTED** [review]
Please remove migrations, table startups and other things from `dev` folder, related to the thing

**Review by srid — APPROVED** [review]
Nix changes approved.

File: Backend/test/src/Mobility/Transporter/APICalls.hs
Comment (hkmangla) [inline Backend/test/src/Mobility/Transporter/APICalls.hs:1]: Instead of removing test cases, lets try reusing them for DriverOfferBPP.

**Review by hkmangla — CHANGES_REQUESTED** [review]

**Comment by 0utkarsh** [comment]
Dhall files can also be removed.

**Review by pavelzarubin — APPROVED** [review]

**Comment by 0utkarsh** [comment]
`Backend/dev/sql-seed/static-offer-driver-app-seed.sql` needs to be deleted too. Can you run the server and check if transporter schema is being created?

**Review by 0utkarsh — APPROVED** [review]

**Comment by 0utkarsh** [comment]
We can remove it from everywhere in Readme as well

**Review by hkmangla — APPROVED** [review]

**Pull request merged**